### DISCUSSION TITLE
【ROOT Android】Steam 3.0 导出令牌的数种方法 🐱 ネコのメモ帳

### DISCUSSION THREAD

**Discussion by giscus** [MuelNova/blog#12] [Announcements]
# 【ROOT Android】Steam 3.0 导出令牌的数种方法 🐱 ネコのメモ帳

在 Steam 更新 3.0 版本之后，原本 2.X 版本下导出令牌的方法已经不再可用。于是我花了一点时间逆向 + Patch，总结出了数个在 Steam 3.0 版本下导出令牌的方法。当然，这些方法的所有前提都是：拥有一台已经 ROOT 过的设备。

https://n.ova.moe/en/blog/2022/11/20/%e3%80%90ROOT%20Android%e3%80%91Steam%203.0%20%e5%af%bc%e5%87%ba%e4%bb%a4%e7%89%8c%e7%9a%84%e6%95%b0%e7%a7%8d%e6%96%b9%e6%b3%95/

**Comment by Z2549** [comment]
https://github.com/YifePlayte/SteamGuardDump
这个开源模块获取后的字段如何在第三方2fa应用中使用，我安装的是aegis，支持添加steam，之前使用电脑的一个软件成功迁移到了aegis并且正常使用。获取的字段为{"accounts":{""{"shared_secret":"","identity_secret":"","secret_1":"","serial_number":"","revocation_code":"","account_name":"","token_gid":"","steamguard_scheme":,"steamid":""}}}
